### ISSUE TITLE
Give Platform Subscribers Monthly Resub Messages

### ISSUE THREAD

**Issue by ILiedAboutCake** [destinygg/website#223]
Similar to the Twitch Experience™ users with a monthly DGG subs should be given the option in their profile to send a TTS message. This should work for monthly resubs as well as people purchasing 3 month subs.

**Comment by georgetdawson** [comment]
Yeah I was thinking about this the other day. Sucks not being able to do that with a dgg sub
